source/services/odnoklassniki.js

```diff
@@ -20,7 +20,7 @@ const odnoklassniki = {
     popupHeight: 400,
 };
 
-set(window, 'ODKL.updateCount', (index, counter) => {
+set(global, 'ODKL.updateCount', (index, counter) => {
```

Comment:
> because when we import package on server, set throws error, because window undefined

Yeah, I get that and I suggest to use global both on server and client. We user commonjs modules so global will be window on client.